files/zh-cn/learn/css/css_layout/positioning/index.html

```diff
@@ -557,7 +560,7 @@ <h6 id="Sticky_positioning_example">Sticky positioning example</h6>
 }
 </pre>
 
-<pre class="notranslate">&lt;h1&gt;Sticky positioning&lt;/h1&gt;
+<pre class="brush:html notranslate">&lt;h1&gt;Sticky positioning&lt;/h1&gt;
```

Comment:
I've added `brush:html` here. If this is missing, then the live sample won't work.